packages/nouns-webapp/src/components/Auction/index.tsx

```diff
@@ -43,16 +43,22 @@ const Auction: React.FC<AuctionProps> = props => {
     currentAuction && history.push(`/noun/${currentAuction.nounId.toNumber() + 1}`);
   };
 
-  const handleKeyPress = useCallback(event => {
-    if (event.code === 'ArrowLeft') prevAuctionHandler();
-    else if (event.code === 'ArrowRight') nextAuctionHandler();
-    else return;
-  }, []);
+  const handleKeyPress = (event: KeyboardEvent) => {
+    if (currentAuction && lastNounId) {
+      const isFirstAuction = currentAuction.nounId.eq(0);
+      const isLastAuction = currentAuction.nounId.eq(lastNounId);
+
+      if (event.code === 'ArrowLeft' && !isFirstAuction) prevAuctionHandler();
```

Comment:
It's also very likely you'll have to wrap `[prev/next]AuctionHandler` in a `useCallback` if they become dependencies of the key press effect.

Reply:
I removed useCallback because it was causing some issues but I'll take another look.